protocol/rtmp/fuzz/fuzz_targets/read_chunks.rs

```diff
@@ -0,0 +1,16 @@
+#![no_main]
```

Comment:
Does this fuzz xiu or rtmp?

Reply:
this is a utility crate that is first-party code to this repo. they don't use vendor sources. If you go to [crates.io](https://crates.io/crates/rtmp) you'll see that they do publish this crate standalone as a library for others to use. But the upstream repo is here.